clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/provider/view/KubernetesClusterProvider.groovy

```diff
@@ -148,6 +147,15 @@ class KubernetesClusterProvider implements ClusterProvider<KubernetesCluster> {
       serverGroup.loadBalancers?.each {
         serverGroup.securityGroups.addAll(securityGroups[it])
       }
+
+      def imageList = new ArrayList()
+      for (def container : serverGroup.deployDescription.containers) {
+        imageList.add(container.imageDescription.repository + ":" + container.imageDescription.tag)
```

Comment:
Could you use this method instead for consistency? https://github.com/spinnaker/clouddriver/blob/master/clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/KubernetesUtil.groovy#L103